docs/conf.py

```diff
@@ -0,0 +1,88 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+import os
+import sys
+from databricks import koala
+sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'databricks-koala'
+copyright = '2019, Timothy Hunter'
+author = 'Timothy Hunter'
+
+# The full version, including alpha/beta/rc tags
+release = os.environ.get('RELEASE_VERSION', koala.__version__)
+
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+needs_sphinx = '1.2'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
```

Comment:
Do you know if there is an extension for rendering jupyter notebooks? I would like to add jupyter notebooks, see #34 . This can be done separately if this is too much effort.

Reply:
Yes, seems we can refer https://nbsphinx.readthedocs.io/en/0.4.2/. Let me add the plugin accordingly for now.